src/backend/people/settings.py

```diff
@@ -334,6 +333,12 @@ class Base(Configuration):
         default=[], environ_name="OIDC_REDIRECT_ALLOWED_HOSTS", environ_prefix=None
     )
 
+    USER_OIDC_FIELDS_TO_NAME = values.ListValue(
+        default=["first_name", "last_name"],
```

Comment:
I have quickly checked the documentation, fyi agent connect exposes `given_name` and `usual_name` in their token I think based [on the documentation ](https://github.com/france-connect/Documentation-AgentConnect/blob/main/doc_fs/projet_fca/projet_fca_donnees.md#les-donn%C3%A9es-obligatoires)

Reply:
Never minds, as these are env values, they can be easily overrode. Nice move

Reply:
nit: `USER_OIDC_FIELDS_TO_NAME` > `OIDC_USER_FIELDS_TO_NAME`

Reply:
ok! I chose the former because I wanted to leave OIDC_* as a namespace for the Mozilla third party application.  But maybe this is not necessary and what you propose is clearer to group it under OIDC matters.